src/Chapter1.hs

```diff
@@ -187,422 +181,419 @@ ghci> :q
-command, and then you should reload it using the ":r" command each time you
-change it and want to check your changes.
--}
-
-{- |
-=⚔️= Task 5
+next x = x + 1
+
+-- |
+-- After you've implemented the function (or even during the implementation), you
+-- can run it in GHCi with your input. To do so, first, you need to load the module
+-- with the function using the ":l" (short for ":load") command.
+--
+-- ghci> :l src/Chapter1.hs
+--
+-- After that, you can call the 'next' function as you already know how to do that.
+-- Or any other function defined in this module! But remember, that you need to
+-- reload the module again after you change the file's content. You can reload the
+-- last loaded module by merely typing the ":r" command (no need to specify the
+-- name again).
+--
+-- ghci> :r
+--
+-- A typical workflow looks like this: you load the module once using the ":l"
+-- command, and then you should reload it using the ":r" command each time you
+-- change it and want to check your changes.n
+
+-- |
+-- =⚔️= Task 5
+--
+-- Implement a function that returns the last digit of a given number.
+--
+-- >>> lastDigit 42
+-- 2
+--
+-- 🕯 HINT: use the `mod` function
+--
+-- ♫ NOTE: You can discover possible functions to use via Hoogle:
+--     https://hoogle.haskell.org/
+--
+--   Hoogle lets you search Haskell functions either by name or by type. You can
+--   enter the type you expect a function to have, and Hoogle will output relevant
+--   results. Or you can try to guess the function name, search for it and check
+--   whether it works for you!
 
-Implement a function that returns the last digit of a given number.
-
->>> lastDigit 42
-2
-
-🕯 HINT: use the `mod` function
-
-♫ NOTE: You can discover possible functions to use via Hoogle:
-    https://hoogle.haskell.org/
-
-  Hoogle lets you search Haskell functions either by name or by type. You can
-  enter the type you expect a function to have, and Hoogle will output relevant
-  results. Or you can try to guess the function name, search for it and check
-  whether it works for you!
--}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
-
-
-{- |
-=⚔️= Task 6
-
-Implement a function, that takes two numbers and returns the one closer to zero:
-
->>> closestToZero 10 5
-5
->>> closestToZero (-7) 3
-3
-
-
-🕯 HINT: You can use the 'abs' function and the __if-then-else__ Haskell syntax
-  for this task.
-
-'if-then-else' is a language construction for expression that returns only one
-branch depending on the checked condition. For example:
-
->>> if even 10 then 0 else 1
-0
-
-The 'if-then-else' constructions must always have both __then__ and __else__
-branches because it is an expression and it must always return some value.
-
-👩‍🔬 Due to lazy evaluation in Haskell, only the expression from the branch
-  satisfying the check will be returned and, therefore, evaluated.
--}
+lastDigit :: Int -> Int
+lastDigit n = mod n 10
```

Comment:
Your implementation is almost correct :ok: 
Unfortunately, it returns negative numbers on negative inputs because of how `mod` works. Sometimes corner cases can be tricky to spot and fix...

Reply:
Right! I've added `abs` to this to handle the edge case